torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1066,7 +1083,11 @@ struct GraphFuser {
       auto subgraph = producer->node()->g(attr::Subgraph);
       auto* node = subgraph->outputs().at(producer->offset())->node();
       return node->kind() != prim::FusedConcat &&
-          !containsGradSumToSize(producer->node());
+          !containsGradSumToSize(producer->node()) &&
+          ((before_check->inputs().size() + before_check->outputs().size() +
+            producer->node()->inputs().size() + producer->node()->outputs().size())
+           <= fusion_kernel_args_limit);
+
```

Comment:
This seems unnecessary as the same check will be performed a few lines later in `tryFuse`, also, if you are trying to fuse concat to something else than a FusionGroup, this function will return `true` (line 1092) which does not look right.

Reply:
Remove this check and let the one in tryFuse() take care of this.